src/main/java/seedu/anilist/ui/StatsDisplay.java

```diff
@@ -57,6 +58,28 @@
     @FXML
     private StackPane statusbarPlaceholder;
 
+    class IntegerStringConverter extends StringConverter<Number> {
+        private long mult = 100000000;
```

Comment:
Maybe use this in conjunction to the method proposed below:
```suggestion
        private final double epsilon = 1E-3;
```

Reply:
thanks I think your method alone is sufficient